homu/server.py

```diff
@@ -419,24 +436,34 @@ def buildbot():
 
 @post('/travis')
 def travis():
+    logger = g.logger.getChild('travis')
+
     info = json.loads(request.forms.payload)
+    logger.debug('info: %s', utils.remove_url_keys_from_json(info))
 
     try: state, repo_label = find_state(info['commit'])
     except ValueError:
-        g.logger.debug('Invalid commit ID from Travis: {}'.format(info['commit']))
+        logger.debug('Invalid commit ID from Travis: {}'.format(info['commit']))
         return 'OK'
 
+    logger.debug('state: %s, %s', state, state.build_res)
```

Comment:
`state` is not printed in `buildbot()`. To keep these consistent, how about adding this to `buildbot()` too?